DESCRIPTION

```diff
@@ -42,6 +42,7 @@ VignetteBuilder: knitr
 BugReports: https://github.com/RforMassSpectrometry/Spectra/issues
 URL: https://github.com/RforMassSpectrometry/Spectra
 biocViews: Infrastructure, Proteomics, MassSpectrometry, Metabolomics
+Remotes: RforMassSpectrometry/MsCoreUtils
```

Comment:
Is that needed?

Reply:
With that we would not need to specifically install `MsCoreUtils` in travis I guess. Ultimately, when all packages are in Bioconductor, we don't need that.

Reply:
Mh, ok. I already added `MsCoreUtils` to `.travis.yml` just to be sure we always use the latest version. So I would like to remove this line so we can't forget it later.

Reply:
OK. Fine with me.